From 05301ce81fa39e531b8324bdf3e7fb21c591564a Mon Sep 17 00:00:00 2001
From: Davis Goodin <dagood@microsoft.com>
Date: Thu, 12 Apr 2018 21:49:01 -0500
Subject: [PATCH] Identity, not Filename, in NugetConfigPrivateFeeds

The key must be unique or NuGet ignores the later feeds. Filename seems indended to make them unique, which doesn't work for source-build because there is no filename for the local directories source-build passes.
---
 build/NugetConfigFile.targets | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/NugetConfigFile.targets b/build/NugetConfigFile.targets
index d1095d7a6..e886b50aa 100644
--- a/build/NugetConfigFile.targets
+++ b/build/NugetConfigFile.targets
@@ -53,7 +53,7 @@
 
     <WriteLinesToFile Condition="'@(NugetConfigPrivateFeeds)' != ''"
                       File="$(GeneratedNuGetConfig)"
-                      Lines="&lt;add key=&quot;PrivateBlobFeed%(NugetConfigPrivateFeeds.Filename)&quot; value=&quot;%(NugetConfigPrivateFeeds.Identity)&quot; /&gt;"
+                      Lines="&lt;add key=&quot;PrivateBlobFeed%(NugetConfigPrivateFeeds.Identity)&quot; value=&quot;%(NugetConfigPrivateFeeds.Identity)&quot; /&gt;"
                       Overwrite="false" />
 
     <WriteLinesToFile Condition="'$(DotNetBuildOffline)' != 'true'"
-- 
2.16.1.windows.4

